python/ray/serve/_private/deployment_state.py

```diff
@@ -1916,6 +1916,10 @@ def update(self) -> Tuple[bool, bool]:
 
             deleted, any_replicas_recovering = self._check_curr_status()
         except Exception:
+            logger.warning(
+                "Exception occurred trying to update deployment state:\n"
+                + traceback.format_exc()
+            )
```

Comment:
you can use `logger.exception` here (this is a system error so error-level is appropriate)